src/helm/benchmark/presentation/run_specs_dec2023.conf

```diff
@@ -0,0 +1,169 @@
+  {description: "mmlu:model=text,subject=conceptual_physics", priority: 4}
+  {description: "mmlu:model=text,subject=electrical_engineering", priority: 4}
+  {description: "mmlu:model=text,subject=elementary_mathematics", priority: 4}
+  {description: "mmlu:model=text,subject=formal_logic", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_biology", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_chemistry", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_computer_science", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_european_history", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_geography", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_government_and_politics", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_macroeconomics", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_mathematics", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_microeconomics", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_physics", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_psychology", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_statistics", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_us_history", priority: 4}
+  {description: "mmlu:model=text,subject=high_school_world_history", priority: 4}
+  {description: "mmlu:model=text,subject=human_aging", priority: 4}
+  {description: "mmlu:model=text,subject=human_sexuality", priority: 4}
+  {description: "mmlu:model=text,subject=international_law", priority: 4}
+  {description: "mmlu:model=text,subject=logical_fallacies", priority: 4}
+  {description: "mmlu:model=text,subject=machine_learning", priority: 4}
+  {description: "mmlu:model=text,subject=management", priority: 4}
+  {description: "mmlu:model=text,subject=marketing", priority: 4}
+  {description: "mmlu:model=text,subject=medical_genetics", priority: 4}
+  {description: "mmlu:model=text,subject=miscellaneous", priority: 4}
+  {description: "mmlu:model=text,subject=moral_disputes", priority: 4}
+  {description: "mmlu:model=text,subject=moral_scenarios", priority: 4}
+  {description: "mmlu:model=text,subject=nutrition", priority: 4}
+  {description: "mmlu:model=text,subject=prehistory", priority: 4}
+  {description: "mmlu:model=text,subject=professional_accounting", priority: 4}
+  {description: "mmlu:model=text,subject=professional_law", priority: 4}
+  {description: "mmlu:model=text,subject=professional_psychology", priority: 4}
+  {description: "mmlu:model=text,subject=public_relations", priority: 4}
+  {description: "mmlu:model=text,subject=security_studies", priority: 4}
+  {description: "mmlu:model=text,subject=sociology", priority: 4}
+  {description: "mmlu:model=text,subject=virology", priority: 4}
+  {description: "mmlu:model=text,subject=world_religions", priority: 4}
+
+  # MATH
```

Comment:
Is it worth including MATH w and w/o COT?

Reply:
and/or should we expand beyond just level 1?

Reply:
I'd just do CoT to keep it simple.  We only had level 1 for classic HELM, so thought we'd just keep it.